bin/create-gxa-analytics-collection.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+SCHEMA_VERSION=1
+
+set -e
+
+# on developers environment export SOLR_HOST_PORT and export SOLR_COLLECTION before running
+HOST=${SOLR_HOST:-"localhost:8983"}
+CORE=${SOLR_COLLECTION:-"gxa-analytics-v$SCHEMA_VERSION"}
+NUMSHARDS=${SOLR_NUM_SHARD:-1}
+REPLICATES=${SOLR_NUM_REPL:-1}
+
+printf "\n\nCreating collection $CORE based on $HOST"
+curl "http://$HOST/solr/admin/collections?action=CREATE&name=$CORE&numShards=$NUMSHARDS&replicationFactor=$REPLICATES"
+
+# Set this value to whatever is needed, it doesn’t really matter with current Lucene versions
+# https://issues.apache.org/jira/browse/SOLR-4586
+MAX_BOOLEAN_CLAUSES=16384
```

Comment:
@alfonsomunozpomer could confirm that we are happy with this max boolean clauses value? I remember that you had to increase it in the past, but maybe that was only applicable to SC?

Reply:
Since I adapted `create-gxa-analytics-collections.sh` from https://github.com/ebi-gene-expression-group/index-scxa I didn't change any parameter/variable values except `SOLR_COLLECTION`

Reply:
No, that’s way too low. Just make it 100,000,000 or some ridiculous number, it has no effect on performance but it will cause unwanted errors. It’s legacy thing.

Reply:
```suggestion
MAX_BOOLEAN_CLAUSES=100000000
```